### ISSUE TITLE
Creating groups on host other than running registry doesn't work

### ISSUE THREAD

**Issue by asheidan** [asheidan/DistSys-Project#2]
```
 [java] java.rmi.ServerException: RemoteException occurred in server thread; nested exception is: 
 [java]     java.rmi.AccessException: Registry.Registry.bind disallowed; origin /130.239.18.213 is non-local host
 [java]     at sun.rmi.server.UnicastServerRef.oldDispatch(UnicastServerRef.java:396)
 [java]     at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:250)
 [java]     at sun.rmi.transport.Transport$1.run(Transport.java:159)
 [java]     at java.security.AccessController.doPrivileged(Native Method)
```

**Comment by asheidan** [comment]
http://stackoverflow.com/questions/820719/using-single-rmi-registry

**Comment by asheidan** [comment]
Working backdoor. This hopefully closed by 0b43cd5c1403bae24485424eec3ddce9f040ac77.
